PyQt5-stubs/QtCore.pyi

```diff
@@ -2263,14 +2263,9 @@ class QIODevice(QObject):
     ExistingOnly = ... # type: 'QIODevice.OpenModeFlag'
 
     class OpenModeFlag(int):
-        def __and__(self, other: typing.Union['QIODevice.OpenModeFlag', 'QIODevice.OpenMode']) -> 'QIODevice.OpenModeFlag': #type: ignore[override]
-            ...
-        def __or__(self, other: typing.Union['QIODevice.OpenModeFlag', 'QIODevice.OpenMode']) -> 'QIODevice.OpenModeFlag': #type: ignore[override]
-            ...
-        def __xor__(self, other: typing.Union['QIODevice.OpenModeFlag', 'QIODevice.OpenMode']) -> 'QIODevice.OpenModeFlag': #type: ignore[override]
-            ...
-        def __ixor__(self, other: typing.Union['QIODevice.OpenModeFlag', 'QIODevice.OpenMode']) -> 'QIODevice.OpenModeFlag': #type: ignore[override]
-            ...
+        def __and__(self, other: typing.Union['QIODevice.OpenModeFlag', 'QIODevice.OpenMode']) -> 'QIODevice.OpenModeFlag': #type: ignore[override] ...
```

Comment:
Looks like the `...`s are after the `#comment`

Reply:
Sorry for the trouble. Fixed. I was being lazy by using Vim's `J` binding for joining both lines.

Reply:
No worries :slightly_smiling_face: